glusterd2/middleware/request_logging.go

```diff
@@ -14,8 +14,20 @@ import (
 // Apache Common Log Format (CLF)
 func LogRequest(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		entry := log.WithField("reqid", gdctx.GetReqID(r.Context()).String())
+		f := log.Fields{
+			"subsys": "rest",
+			"reqid":  gdctx.GetReqID(r.Context()).String(),
+		}
+		if origin := r.Header.Get("X-Gluster-Origin-Args"); origin != "" {
+			f["origin"] = origin
```

Comment:
TRIVIAL: IMO origin seems a bit ambiguous, a better variable name can be chosen, something like orgnArgs.

Reply:
Sounds good.